pages/docs/how-to-guides/links.mdx

```diff
@@ -56,8 +56,50 @@ To add more links add another object inside the links collection. For example:
 
 _Clicks on these social shortcuts that match any of your links will also increment their respective counters_
 
-### What Links with grouping looks like
+### Grouping your Links 
```

Comment:
@CBID2 What do you think about reiterating here either in the title or description that this is an optional feature.  Some folks may just scroll quickly and miss that this is optional farther up

Reply:
Ooh good idea!